.github/workflows/tf-feature.yaml

```diff
@@ -18,7 +18,10 @@ on:
         description: "The AWS account ID."
         type: string
       gh_artifact_path:
-        description: "Path to download artifacts to. If not set, no artifacts will be downloaded."
+        description: "Path to download artifacts to."
+        type: string
+      gh_artifact_name:
+        description: "Name of the artifact to download. If only 'gh_artifact_path' is set, then all artifacts are downloaded."
```

Comment:
adding this can make it a little confusing as to whats possible. Should we just add a gh_artifacts_enable flag?

Reply:
I thought of it @swibrow, but couldn't find a design I liked so much (considering we'd be introducing breaking changes). I'd rather introduce another input and let the behaviour be exactly the same as the one upstream (of the `download-artifacts` action) without any modifications. 

In a sense we are just passing upstream config, makes sense for me to not modify the behaviour with additional config (a bool). If the bool is true and the other values not set (for example), what do we do?

Reply:
Ok, missed that. I thought at least one of the two would be mandatory. 

Then it's a breaking change, but makes sense 😅  v2?